pkg/pluginmanager/manager.go

```diff
@@ -697,24 +697,28 @@ func GetPluginGroup(groupIDAndVersion string, options ...PluginManagerOptions) (
 	return pg, nil
 }
 
-func logPluginInstallationMessage(p *discovery.Discovered, version string, isPluginInCache, isPluginAlreadyInstalled bool) {
+func getPluginInstallationMessage(p *discovery.Discovered, version string, isPluginInCache, isPluginAlreadyInstalled bool) (installingMsg, installedMsg string) {
 	withTarget := ""
 	if p.Target != configtypes.TargetUnknown {
-		withTarget = fmt.Sprintf("with target '%v' ", p.Target)
+		withTarget = fmt.Sprintf("with target '%v'", p.Target)
 	}
 
 	if isPluginInCache {
 		if !isPluginAlreadyInstalled {
-			log.Infof("Installing plugin '%v:%v' %v(from cache)", p.Name, version, withTarget)
+			installingMsg = fmt.Sprintf("Installing plugin '%v:%v' %v (from cache)", p.Name, version, withTarget)
+			installedMsg = fmt.Sprintf("Installed plugin '%v:%v' %v (from cache)", p.Name, version, withTarget)
 		} else {
-			log.Infof("Plugin '%v:%v' %vis already installed. Reinitializing...", p.Name, version, withTarget)
+			installingMsg = fmt.Sprintf("Plugin '%v:%v' %v is already installed. Reinitializing...", p.Name, version, withTarget)
+			installedMsg = fmt.Sprintf("Reinitialized plugin '%v:%v' %v", p.Name, version, withTarget)
```

Comment:
Nit: there will be an empty space at the end of the string if the `withTarget` string is empty.
You can fix this by adding a space to the start of `withTarget` instead of at the end, and adjusting the other strings.

Reply:
Agree, fixed.